src/web/routes/application/steps/decision/get-decision-status.js

```diff
@@ -0,0 +1,26 @@
+const { equals, compose, prop, cond, always } = require('ramda')
+
+const SUCCESS = 'success'
+const FAIL = 'fail'
+const PENDING = 'pending'
+
+const outcomeNotMatched = equals('not_matched')
+const outcomeNotConfirmed = equals('not_confirmed')
+
+const identityOutcomeNotMatched = compose(outcomeNotMatched, prop('identityOutcome'))
+const eligibilityOutcomeNotConfirmed = compose(outcomeNotConfirmed, prop('eligibilityOutcome'))
+
+const isFailure = result => identityOutcomeNotMatched(result) || eligibilityOutcomeNotConfirmed(result)
+
+const getDecisionStatus = cond([
+  [isFailure, always(FAIL)]
+])
```

Comment:
`cond` facilitates mapping predicates to decision status. When all scenarios are implemented it should hopefully look like (returns `undefined` by default):
```
const getDecisionStatus = cond([
  [isFailure, always(FAIL)],
  [isSuccess, always(SUCCESS)],
  [isPending, always(PENDING)]
])
```